src/encoded/static/libs/formatMeasurement.js

```diff
@@ -0,0 +1,26 @@
+
+/**
+* Takes a magnitude and unit (singular form) and returns the measurement
+* with the proper unit in singular or plural, depending on magnitude.
+* Invalid measurements are turned as- Unknown
+* @param {number} magnitude - Measured value
+* @param {string} unit - Measurement system in singular form
+*/
+export default function formatMeasurement(magnitude, unit) {
+    // parseFloat used because parseInt only keeps integer
+    const magnitudeAsFloat = parseFloat(magnitude);
+
+    if (Number.isNaN(magnitudeAsFloat)) {
+        return 'Unknown';
+    }
+
+    if (magnitudeAsFloat === 0) {
+        return '0';
+    }
+
+    const measurement = `${magnitudeAsFloat} ${unit || ''}`.trim();
+
+    return magnitudeAsFloat === 1
+          ? measurement
+          : `${measurement}s`;
+}
```

Comment:
AirBnB prefers the:

```
const formatMeasurement = function formatMeasurement() {…};
export default formatMeasurement;
```

…syntax (https://github.com/airbnb/javascript#functions). But two things about that:

1. The eslint editor plugin with AirBnB extension doesn’t call this out (probably their bug).
2. This conflicts with Redux which prefers:

```
const formatMeasurement = (magnitude, unit) => {
};
export default formatMeasurement;
```

I prefer the Redux preference lately, but I think for now this is all programmer’s discretion. I do see the reasoning for AirBnB’s opinion, and I might go back to that. I have no issue with inconsistencies with this, personally.